GitVersion.yml

```diff
@@ -26,7 +26,7 @@ branches:
     is-release-branch: false
   hotfix:
     regex: ^hotfix(es)?[/-]
-    tag: beta
+    tag: useBranchName
```

Comment:
I think using branch name makes more sense for hotfixes, else the package is always suffixed with 'beta'